report-summary-merged-prs

```diff
@@ -1610,6 +1616,8 @@ if __name__ == "__main__":
                     find_general_test_results(
                         'reco_event_loop', release_queue_results['comparisons'], arch, MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER2, find_one_profiling_result
                         )
+                if 'ubsan-logs' in tests_to_find:
+                    find_ubsan_logs(release_queue_results['comparisons'], arch)
```

Comment:
remove it from here and add it unde under https://github.com/cms-sw/cms-bot/blob/ae18ea0f7f11421d752ca28cdfd104a9c2d0ee24/report-summary-merged-prs#L1645 . No need to add the additional test in config.map file